.dockerignore

```diff
@@ -0,0 +1,46 @@
+.git/
+*~
+*.bak
+__pycache__
+opentitan-docs
+
+# fusesoc result
+build/
+
+# build directories with special suffixes
+build*/
+
+# Sphinx result
+_build/
```

Comment:
Where is that coming from?

Reply:
... Maybe the buildthedocs trial? I will remove this line :)